pdaltools/las_info.py

```diff
@@ -19,6 +21,44 @@ def las_info_metadata(filename: str):
         raise RuntimeError(f"pdal info failed with error: \n {ret.stderr}")
 
 
+def las_info_from_header(filename: str):
+    r = pdal.Reader.las(filename=filename)
+    p = r.pipeline()
+    metadata = p.quickinfo["readers.las"]
+
+    return metadata
+
+
+def get_bounds_from_header_info(metadata):
+    bounds = metadata["bounds"]
+    minx, maxx, miny, maxy = bounds["minx"], bounds["maxx"], bounds["miny"], bounds["maxy"]
+
+    return minx, maxx, miny, maxy
+
+
+def get_epsg_from_header_info(metadata):
+    if "srs" not in metadata.keys():
+        raise RuntimeError(
+            "EPSG could not be inferred from metadata, please provide a 'proj' value in the 'color' method."
+        )
+
+    else:
+        proj = metadata["srs"]
+        # use compoundwkt as recommended in https://github.com/PDAL/python/issues/112
+        wkt = proj["compoundwkt"]
+        osr_crs = SpatialReference()
+        osr_crs.ImportFromWkt(wkt)
+        authority = osr_crs.GetAttrValue("AUTHORITY", 0)
+        if authority == "EPSG":
+            proj = osr_crs.GetAttrValue("AUTHORITY", 1)
+        else:
+            raise RuntimeError(
+                "EPSG could not be inferred from metadata, please provide a 'proj' value in the 'color' method."
+            )
```

Comment:
Pareil:
"EPSG could not be inferred from metadata: no attribute EPSG found in metadata srs"